src/main/java/org/rumbledb/compiler/StaticContextVisitor.java

```diff
@@ -288,4 +289,18 @@ public StaticContext visitTypeSwitchExpression(TypeSwitchExpression expression,
     }
     // endregion
 
+    // region FLWOR vars
```

Comment:
I think this region specifier may have been accidentally copied.

Reply:
Done